index.html

```diff
@@ -180,40 +180,44 @@ <h2>Contact me</h2>
               <input
                 type="text"
                 name="name"
-                placeholder="Aimable"
+                id="name"
+                placeholder="Name"
                 maxlength="30"
                 required
               />
             </li>
             <li class="li1-li_st">
               <input
                 type="email"
-                name="email"
-                placeholder="bancunguye@gmail.com"
+                name="Email"
+                id="email"
+                pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
+                placeholder="Email"
                 required
               />
             </li>
             <li class="li2-li_st">
               <textarea
                 name="comments"
-                placeholder="comment"
+                placeholder="Write a message"
                 maxlength="500"
+                id="textarea"
                 required
               ></textarea>
             </li>
-            <li class="li3-li_st">
-              <input
-                class="btn-btn-default"
-                type="submit"
-                name="submit"
-                value="Get in touch"
-              />
-            </li>
           </ul>
+          <div class="contact_button">
+            <small id="alertmessage"></small>
+            <button id="contact_button" type="submit">Get In Touch</button>
+          </div>
+          <div class="submit_response error_hidden">
+            Thank you for your Message
+          </div>
         </form>
```

Comment:
- [ ] [**OPTIONAL**]Good job working consistently on your portfolio, however sticking to the Figma is an important part of the requirements :+1: , Kindly remove this padding on the bottom of your page to meet the design required :+1: 

![Screenshot from 2023-03-10 19-43-06](https://user-images.githubusercontent.com/80458199/224387096-f8c7d7ba-1b5e-4022-8cce-36660a0fef7d.png)